src/planscape/conditions/load.py

```diff
@@ -2,6 +2,7 @@
 from pathlib import Path
 from decouple import config
 from typing import cast
+import sys
```

Comment:
this file has some of my unfinished changes and i'm not really sure how that happened! maybe we can exclude it for now?

Reply:
I'll fix in a follow-up.